google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -200,15 +201,19 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
+  std::mutex mu;
```

Comment:
Nit: Perhaps something like `// serialize output` might clarify this (assuming I understand it).